src/chi-vue/src/components/data-table/DataTableUtils.ts

```diff
@@ -0,0 +1,18 @@
+export function printDataTable(content: string, title: string) {
```

Comment:
Can we make the function element-agnostic? We might need the util function for some other elements as well in future.
Can we move it to generic `utils` folder and rename it to `printElement`?

Reply:
+1

Reply:
Yes, makes sense. The utility function will be Chi Vue specific.